app/components/main-screen.js

```diff
@@ -1,4 +1,9 @@
 import Component from '@ember/component';
+import { A } from '@ember/array';
 
 export default Component.extend({
+  orderActions: A([
+    { icon: 'expand_less' },
+    { icon: 'expand_more' }
```

Comment:
@einnor I would like to add name: 'asc' and name 'desc' as well here, if it makes sense with the selection of the chevron.